Remotion/Web/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRenderer.cs

```diff
@@ -171,7 +171,7 @@ private void RenderListMenuItem (ListMenuRenderingContext renderingContext, WebM
         attributes.Add ("tabindex", isFirst ? "0" : "-1");
 
 
-      var command = !menuItem.IsDisabled ? menuItem.Command : new Command (CommandType.None) { OwnerControl = menuItem.OwnerControl };
+      var command = !menuItem.IsDisabled ? menuItem.Command! : new Command (CommandType.None) { OwnerControl = menuItem.OwnerControl }; // TODO: not null assertion for command
```

Comment:
I'm not sure if this would be a good move, but what about making `WebMenuItem.Command` non-null and using `new Command(CommandType.None)` as default? (in a separate issue)

Reply:
perhaps. I don't want to lock in a solution at this time so a non-specific cleanup-task that will also collect the suggestions? This requires a bit of reverse engineering about what's intentional, what's just for the designer (so...dead), and what's stupid.